### PR TITLE
Added (theta, mag) representation to PointScatter.java

### PR DESCRIPTION
**Pull request by JavaNoTea**
- Added parameter 'data representation' so we can choose between the xy Cartesian and the theta mag polar representations
- Fixed the directory name on the MakeFile
- Added functionality for the theta mag polar representation
- Reformatted PointScatter so that both representations would work

### CODE REVIEW

File: src/Makefile [hunk 1]
```diff
@@ -2,7 +2,7 @@ build:
 	javac *.java
 
 run: build
-	java Search parameters/pointscatter.params
+	java Search params/pointscatter.params
```
Comment (andey-robins) [inline src/Makefile:5]: Whoops, thanks for this catch.

File: src/PointScatter.java [hunk 2]
```diff
@@ -9,34 +9,62 @@ public PointScatter() {
...
-                int x2 = X.getPosIntGeneValue(j);
-                int y2 = X.getPosIntGeneValue(j + 1);
-                double d = distanceBetweenPoints(x1, y1, x2, y2);
-                if (d < X.rawFitness) {
-                    X.rawFitness = d;
-                }
-            }
+        
+        
+        if (Parameters.dataRepresentation.equals("xycart")){
```
Comment (andey-robins) [inline src/PointScatter.java:25]: The two options we have, "xycart" and "thetamagpolar" should make their way to our README file. I just want to note this down for future modifications and having them explicitly listed here will save me time when I do those docs.

File: src/params/pointscatter.params [hunk 3]
```diff
@@ -1,4 +1,5 @@
 Experiment ID                :pointscatter
+Data Representation          :thetamagpolar
```
Comment (andey-robins) [inline src/params/pointscatter.params:2]: I think we can even make a couple parameter files, I'll make sure subsequent additions have different names than this one.

**Review by andey-robins — CHANGES_REQUESTED** [review]
LGTM, can we just do a quick rebase and squash the commits since each commit is only touching one file?

**Comment by JavaNoTea** [comment]
Squashed the commits

**Comment by andey-robins** [comment]
Awesomesauce! Thanks for the squash! Just merged in

**Pull request merged**